test/e2e_node/e2e_node_suite_test.go

```diff
@@ -85,7 +85,7 @@ func registerNodeFlags(flags *flag.FlagSet) {
 	// It is hard and unnecessary to deal with the complexity inside the test suite.
 	flags.BoolVar(&framework.TestContext.NodeConformance, "conformance", false, "If true, the test suite will not start kubelet, and fetch system log (kernel, docker, kubelet log etc.) to the report directory.")
 	flags.BoolVar(&framework.TestContext.PrepullImages, "prepull-images", true, "If true, prepull images so image pull failures do not cause test failures.")
-	flags.BoolVar(&framework.TestContext.RestartKubelet, "restart-kubelet", true, "If true, restart Kubelet unit when the process is killed.")
+	flags.BoolVar(&framework.TestContext.RestartKubelet, "restart-kubelet", false, "If true, restart Kubelet unit when the process is killed.")
```

Comment:
why this change?

Reply:
Now that we do not have DynamicKubelet configuration, we do not need to run a restart loop.

How did it work before:
1. Test updated kubelet via DynamicKubeletConfiguration
2. DynamicKubeletConfiguration controller applied updated configuration and stopped the kubelet.
3. The restart loop monitors the kubelet health link and once it does not respond restart the kubelet.

How does it work now:
1. The test stops the kubelet.
2. The test update kubelet static configuration.
3. The test restarts kubelet.

So now it totally tests responsibility to make sure the kubelet runs after the test. It makes restarts more predictable because it is not external to the test component that can restart the kubelet in the middle of the update configuration.

Reply:
I'm thinking if there were other uses like when kubelet paniced or something. I don't remember any tests explicitly stopping kubelet. But I saw supressed panics before (https://github.com/kubernetes/kubernetes/pull/101719). 

I tend to ok this change since panic is bad enough condition to make test grid red.